### PR TITLE
Rename epic pools

### PR DESCRIPTION
**Pull request by johnlamping**
Everywhere in the UI, change
EP -> PWR
SP -> ER

### CODE REVIEW

**Comment by johnlamping** [comment]
This only changes the appearance of the UI. It doesn't rename the variables in the script. Doing that is tricky, because you don't want to break all the tokens that reference variables by their current name. The right thing, I think, is to put triggers in the scripts that copy data between the new and old names, so that tokens will work either way. That is more than I want to do right now.

File: ui/rollSection/rollSection.pug [hunk 1]
```diff
@@ -34,11 +34,11 @@
       input#attr_action_description.flexing(name='attr_action_description' type='text')
       label(for='attr_action_feats') Feats
       input#attr_action_feats.narrowish(name='attr_action_feats' type='number' value='0')
-      label(for='attr_action_feats_EP') EP &ensp;
+      label(for='attr_action_feats_EP') PWR &ensp;
       input#attr_action_feats_EP.narrow(name='attr_action_feats_EP' value='@{action_feats}' disabled='true')
-      label(for='attr_action_EP') EP
+      label(for='attr_action_EP') PWR
```
Comment (jazeee) [inline ui/rollSection/rollSection.pug:39]: The inner variables will be a bit of a pain (`attr_action_EP`, etc.)
I don't know what we should do with that.
Maybe we should follow up with a rename and migration to clone the values to a matching name...

**Review by jazeee — APPROVED** [review]
Looks good
🚀 🚀 🚀

File: ui/rollSection/rollSection.pug (hunk 1 above)
Comment (johnlamping) [reply ui/rollSection/rollSection.pug:39]: That sounds right.

I think it requires three things:
- Code at character sheet open that copies the values under the old names to the new ones.
- Triggers on changes to the old variables that copy to the new ones
- Triggers on changes to the new variables that copy to the old ones.

The last two are so all the character tokens that refer to the old variables will keep working.

**Pull request merged**